Source/UI.wl

```diff
@@ -546,31 +518,14 @@ chatRequest[messages_, tokenLimit_, temperature_] := Module[{apiKey},
 	If[!StringQ[apiKey],
 		Raise[
 			ChatbookError,
-			<| "SystemCredentialKey" -> $openAICredentialKey |>,
-			"unexpected result getting OpenAI API Key from SystemCredential: ``",
+			<| "SystemCredentialKey" -> ChatServiceData[service,"AuthKeyName"],
+				"ServiceName" -> ChatServiceData[service,"ServiceName"] |>,
+			"unexpected result getting `ServiceName` API Key from SystemCredential: `SystemCredentialKey`",
```

Comment:
This syntax of using named template parameters isn't supported by `Raise` (though perhaps it should be).